hw09-jdbc/src/main/java/ru/otus/lantukh/jdbc/mapper/EntityClassMetaDataImpl.java

```diff
@@ -0,0 +1,125 @@
+package ru.otus.lantukh.jdbc.mapper;
+
+import ru.otus.lantukh.core.model.Id;
+
+import java.lang.reflect.Constructor;
+import java.lang.reflect.Field;
+import java.math.BigInteger;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class EntityClassMetaDataImpl<T> {
```

Comment:
Надо реализовать интерфейс.
```suggestion
public class EntityClassMetaDataImpl<T> implements  EntityClassMetaData<T> {
```